### PR TITLE
Cht add pcolormesh

### PR DESCRIPTION
**Pull request by jklymak**
`pcolormesh` does not appear in the cheatsheet, yet it is often more useful than `imshow`.  Suggest replacing `xcorr` (which is pretty obscure, and plotted in a digital manner, which is not how I've ever seen it done) with `pcolormesh`.  One could argue that adding a `stem` plot instead of `xcorr` may be helpful, however if we did that I'd probably get rid of `eventplot`.

### CODE REVIEW

**Comment by jklymak** [comment]
[cheatsheets.pdf](https://github.com/matplotlib/cheatsheets/files/7538889/cheatsheets.pdf)
cheatsheets copy

**Comment by jklymak** [comment]
I don't really understand why the trim trailing whitespace is failing...

**Comment by jimustafa** [comment]
> I don't really understand why the trim trailing whitespace is failing...

There are 4 space characters in line 14. Might have snuck in when pre-commit checks were not looking...
https://github.com/matplotlib/cheatsheets/blob/297ef35b0228879c27d21046ff33bbae10d4d1ad/.circleci/config.yml#L1-L18

File: cheatsheets.tex [hunk 1]
```diff
@@ -382,6 +382,15 @@
          \optional{extent},
          \optional{origin} }
        {}
+  \plot{basic-pcolormesh.pdf}{\textbf{pcolormesh[f]}([X],[Y],Z,,…)}
```
Comment (timhoffm) [inline cheatsheets.tex:385]:
```suggestion
  \plot{basic-pcolormesh.pdf}{\textbf{pcolormesh[f]}([X],[Y],Z,…)}
```

Comment (timhoffm) [reply cheatsheets.tex:385]: This is a copy-paste error from contour. While at it remove the second comma there as well.

**Review by timhoffm — COMMENTED** [review]
I agree that pcolormesh is more relevant than xcorr.

File: cheatsheets.tex (hunk 1 above)
Comment (jklymak) [reply cheatsheets.tex:385]: ooops!  Fixed

**Review by timhoffm — APPROVED** [review]

**Pull request merged**